server/plugin/command.go

```diff
@@ -531,9 +531,18 @@ func (p *Plugin) isAuthorizedSysAdmin(userID string) (bool, error) {
 }
 
 func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
-	command, action, parameters := parseCommand(args.Command)
+	cmd, action, parameters := parseCommand(args.Command)
 
-	if command != "/github" {
+	if cmd != "/github" {
+		return &model.CommandResponse{}, nil
+	}
+
+	if action == "about" {
```

Comment:
Since `info` is already used in other plugin projects, and has been used as a tool with the support team to gather information plugin installations, 2/5 I'm thinking we should use `info` here

Reply:
>  has been used as a tool with the support team to gather information plugin installations,

The only plugin used in production that uses an `info` command is Jira, right? Even that command works slightly differently. I would like for the `about` command to not include any confidential information, so it can be used for debugging without manual cleanup.

Reply:
@mickmister What do you think about my comment?